src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -236,12 +236,21 @@ private IList<string> GetResourceNamesForCulture(CultureInfo culture)
             {
                 var names = new List<string>();
```

Comment:
You could declare this right above the `foreach` and return it from there. Does't seem like it's used outside of the loop.